test/FadeMixinSpec.jsx

```diff
@@ -36,24 +36,27 @@ describe('FadeMixin', function () {
     }, 25)
   });
 
-  it('Should remove the in class for all elements', function (done) {
-    var instance = ReactTestUtils.renderIntoDocument(<Component />);
-
-    setTimeout(function(){
-      instance.componentWillUnmount()
-      var element = instance._fadeOutEl.children[0]
-      var child   = element.children[0]
-
-      assert.ok(element.className.match(/\bin\b/));
-      assert.ok(child.className.match(/\bin\b/));
-
-      setTimeout(function(){
-        assert.ok(!element.className.match(/\bin\b/));
-        assert.ok(element.className.match(/\bfade\b/));
-        assert.ok(!child.className.match(/\bin\b/));
-        assert.ok(child.className.match(/\bfade\b/));
-        done()
-      }, 25)
-    }, 25)
-  });
+  // it.only('Should remove the in class for all elements', function (done) {
```

Comment:
I'd prefer to see this fixed before we merge this. I'm not a fan if intentionally bringing in commented out code.